nautobot/dcim/tests/test_api.py

```diff
@@ -2904,3 +2907,46 @@ def setUpTestData(cls):
         cls.bulk_update_data = {
             "weight": 300,
         }
+
+
+class ModuleTestCase(APIViewTestCases.APIViewTestCase):
+    model = Module
+
+    @classmethod
+    def setUpTestData(cls):
+        mt = ModuleType.objects.create(
+            manufacturer=Manufacturer.objects.first(),
+            model="Test Module Type",
+        )
+        module_bay = ModuleBay.objects.create(
+            parent_device=Device.objects.first(),
+            position="0",
+        )
```

Comment:
can we factory-ize these?

Reply:
There are already factories for these. This setup data will likely be removed when I make it back around to working on the rest api after I finish the filters.